### PR TITLE
Added Unit Tests for all 3 charms

### PR DESCRIPTION
**Pull request by sachin-trilio**
1. Added unit tests for - charm-trilio-data-mover-api and charm-trilio-data-mover
2. Updated charm-trilio-horizon-plugin tests
3. Updated TODO.txt

### CODE REVIEW

File: juju-charms/charm-trilio-data-mover-api/unit_tests/test_charm_openstack_dmapi.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        dmapi.render_configs('interfaces-list')
+        self.render_with_interfaces.assert_called_once_with(
+            'interfaces-list')
+
+class TestDmapiAdapters(Helper):
+
+    @mock.patch('charmhelpers.core.hookenv.config')
+    def test_dmapi_adapters(self, config):
+        reply = {
+            'keystone-api-version': '2',
```
Comment (abhijeetpatra) [inline juju-charms/charm-trilio-data-mover-api/unit_tests/test_charm_openstack_dmapi.py]: should it be 3 ?

Comment (sachin-trilio) [reply juju-charms/charm-trilio-data-mover-api/unit_tests/test_charm_openstack_dmapi.py]: Done. Updated to 3.

**Comment by MuralidharB** [comment]
@sachin-trilio can you resolve the conflicts so I can merge the PR.

**Comment by sachin-trilio** [comment]
Resolved the conflicts.

**Pull request merged**